src/managers/KeybindManager.cpp

```diff
@@ -2177,7 +2177,12 @@ void CKeybindManager::moveOutOfGroup(std::string args) {
     if (!*PIGNOREGROUPLOCK && g_pKeybindManager->m_bGroupsLocked)
         return;
 
-    const auto PWINDOW = g_pCompositor->m_pLastWindow;
+    CWindow* PWINDOW = nullptr;
+
+    if (args != "" && args != "active" && args.length() > 1)
```

Comment:
args.empty() not != "", but you already check for .length() so why is it there even

Reply:
That's a good question. I wondered the same. The other two implementation did the same thing so I figured there must have been a good reason and just left it there

Reply:
Fixed it to use the empty and also did the same with the other implementations